### DISCUSSION TITLE
Is it possible to remove top and bottom window margins?

### DISCUSSION THREAD

**Discussion by VKondakoff** [kovidgoyal/kitty#4134] [Q&A] [answered]
Hi!

Is there a way to remove top and bottom window margins?
<img width="617" alt="Снимок экрана 2021-10-18 в 18 25 19" src="https://user-images.githubusercontent.com/230824/137762115-b68c671e-dcf5-4c5e-9a86-1348cb87a6f7.png">
Thanks in advance!

**Comment by kovidgoyal** [comment, marked as answer]
Make the window size an exact multiple of the cell size
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.remember_window_size
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.resize_in_steps

**Reply by VKondakoff** [reply]
That's it! Thanks!

**Reply by page-down** [reply]
If you need a fixed window size. You can set it as a shortcut key and it will resize directly to the desired size when pressed.

For example, execute the following command to start a demo instance and press the `F1`, which will resize the os window.
```shell
kitty -o 'map f1 remote_control resize-os-window --width=80 --height=24'
```
